src/platforms/stlinkv3/README.md

```diff
@@ -49,7 +51,7 @@ It is a good idea to keep a full image of the original flash content as backup!
 If you want to keep the original bootloader or access via SWD is disabled, clone
 https://github.com/UweBonnes/stlink-tool/tree/stlinkv21
```

Comment:
Same comment on the URL needing updating.

Reply:
Fixed.